src/vs/workbench/services/gettingStarted/common/gettingStartedContent.ts

```diff
@@ -42,141 +39,195 @@ type GettingStartedContent = GettingStartedCategory[];
+					button: {
+						title: localize('gettingStarted.openVSC.button', "Open in VS Code"),
+						command: 'github.codespaces.openInStable'
+					},
+					doneOn: { commandExecuted: 'github.codespaces.openInStable' },
+					media: { type: 'image', altText: 'Preview of the Open in VS Code command.', path: 'openVSC.jpg' },
+				}
+			]
+		}
+	},
 
+	{
+		id: 'Setup',
+		title: localize('gettingStarted.setup.title', "Quick Setup"),
+		description: localize('gettingStarted.setup.description', "Extend and customize VS Code to fit your needs."),
+		icon: setupIcon,
+		when: 'emptyWorkspaceSupport',
+		content: {
+			type: 'items',
+			items: [
+				{
+					id: 'pickColorTheme',
+					title: localize('gettingStarted.pickColor.title', "Customize the Look With Themes"),
+					description: localize('gettingStarted.pickColor.description', "Adapt VS Code to your taste with themes, customizing interface and language syntax colors."),
+					button: { title: localize('gettingStarted.pickColor.button', "Browse Color Themes"), command: 'workbench.action.selectTheme' },
+					doneOn: { eventFired: 'themeSelected' },
+					media: { type: 'image', altText: 'Color theme preview for dark and light theme.', path: 'colorTheme.jpg', }
+				},
 				{
 					id: 'findLanguageExtensions',
-					description: localize('findLanguageExtsTask.description', "Get support for your languages like JavaScript, Python, Java, Azure, Docker, and more."),
-					title: localize('findLanguageExtsTask.title', "Languages & Tools"),
+					title: localize('gettingStarted.findLanguageExts.title', "Add More Language & Tools Support"),
+					description: localize('gettingStarted.findLanguageExts.description', "Install extensions with one click to support additional languages like Python, Java, Azure, Docker, and more."),
 					button: {
-						title: localize('findLanguageExtsTask.button', "Install Language Support"),
+						title: localize('gettingStarted.findLanguageExts.button', "Browse Language Extensions"),
 						command: 'workbench.extensions.action.showLanguageExtensions',
 					},
 					doneOn: { commandExecuted: 'workbench.extensions.action.showLanguageExtensions' },
-					media: { type: 'image', altText: 'Languages', path: 'Languages.jpg', }
+					media: { type: 'image', altText: 'Language extensions', path: 'languageExtensions.jpg', }
+				},
+				{
+					id: 'githubLogin',
+					title: localize('gettingStarted.githubLogin.title', "Use GitHub in VS Code"),
+					description: localize('gettingStarted.githubLogin.description', "Integrated GitHub makes it easier for you to manage projects from inside your code editor, including authentication, publishing repos, and viewing your repo timeline."),
+					when: '!githubBrowser:hasProviders',
+					button: {
+						title: localize('gettingStarted.githubLogin.button', "Sign in to GitHub"),
+						command: 'githubsignIn',
```

Comment:
command seems to not exist? cc @RMacfarlane is there a command for this in general or would it need to be triggered by an extension requesting specific scopes?

Reply:
There isn't a command for this in general - I think we would want to only log in for a specific purpose. Logging in does require requesting specific scopes, and since we don't have a way to exchange GitHub tokens for tokens with fewer permissions, asking a user to sign in now could still trigger the log in flow later.